arangod/RestHandler/RestImportHandler.cpp

```diff
@@ -1000,6 +1000,13 @@ Result RestImportHandler::performImport(SingleCollectionTransaction& trx,
     res = opResult.result;
   }
 
+  if (collectionName == "_appbundles") {
```

Comment:
Why is it _appbundles here? and _apps below?

I think we need to trigger it if either one is imported:
```suggestion
  if (collectionName == "_appbundles" || collectionName == "_apps") {
```

Reply:
Yes, we could do it like this. But we should discuss further. If that is the right way to do it.